e2e/reporters/reporter.js

```diff
@@ -48,14 +48,12 @@ class SnapshotAwareReporter {
     const filename = path.basename(testPath);
     const name = ancestorTitles.join(' > ');
     const statBase = {name, filename, status, title, time: duration};
-
     if (status === 'passed') return statBase;
 
-    const snapshotParts = [filename, ...ancestorTitles, title, '1-snap', 'diff'];
-    const snapshotName = snapshotParts.map(sanitize).join('-') + '.png';
+    const snapshotParts = [filename, ...ancestorTitles, title, 'diff'];
```

Comment:
As was mentioned before we use full test path and test full name (it is presented in the test object but probably it just called differently from the `currentTestName`, so that will be the only difference of how we pass input to blame builder)